apps/fem/poisson3.sim

```diff
@@ -0,0 +1,188 @@
+%
+%  Strong form (poisson problem)
+%
+% - \nabla^2 u = f(x,y) in \Omega (domain)
+%            u = 0      on \Gamma (boundary)
+%
+%  solved using linear functions on a tetrahedral mesh
+%
+
+element Node
+    x    : vector[3](float); % position
+    edge : bool;             % true if part of the boundary
+    u    : float;            % solution
+    i    : int;              % index of node
```

Comment:
Just for my edification, is there a reason why the tet version needs `i` but not the tri version?